src/com/sun/ts/tests/ejb/ee/bb/session/stateless/bm/allowedmethodstest/TestBeanEJB.java

```diff
@@ -508,19 +508,21 @@ private void doOperationTests(String s) {
     }
 
     // getMessageContext test
-    try {
-      sctx.getMessageContext();
-      TestUtil.logMsg("Operations test: getMessageContext() - allowed");
-    } catch (IllegalStateException e) {
-      methodList[i].setProperty("getMessageContext", "false");
-      TestUtil.logMsg("Operations test: getMessageContext() - not allowed");
-    } catch (Exception e) {
-      TestUtil.printStackTrace(e);
-      methodList[i].setProperty("getMessageContext", "unexpected");
-      TestUtil.logMsg(
-          "Operations test: getMessageContext() - not allowed (Unexpected Exception) - "
-              + e);
-    }
+// TODO: Verify
```

Comment:
What should we do with test code for pruned technologies like JAX-RPC?  I'll send email about this one.

Reply:
https://github.com/eclipse-ee4j/jakartaee-tck/issues/184 - it's documented to not forget about it.